### DISCUSSION TITLE
Does Greed have a french localization?

### DISCUSSION THREAD

**Discussion by UHQNYZOO91** [Steffo99/greed#181] [Q&A] [answered]
Hello du you have version french ? pls

**Comment by Steffo99** [comment, marked as answer]
[As of 2022-07-12, no French localization has been contributed yet.](https://github.com/Steffo99/greed/tree/8701071b30246c0046ffd4b42037c9c335dd0377/strings)

If you speak French, you can create one:

1. Copy one of the pre-existing localization files to a new file named `fr.py`
2. Edit all strings with your desired translation
3. Add the string `"fr"` to the `enabled_languages` list of your config
4. Optionally, contribute the localization via a [pull request](https://github.com/Steffo99/greed/pulls) so others may make use of it
